src/ports/postgres/modules/deep_learning/madlib_keras.py_in

```diff
@@ -476,15 +470,14 @@ def fit_transition(state, dependent_var, independent_var, model_architecture,
         total_images = images_per_seg[0]
     else:
         total_images = images_per_seg[seg_ids.index(current_seg_id)]
+
     if total_images == 0:
-        plpy.error('Got 0 rows. Expected at least 1.')
+        plpy.error('Total images is 0 in fit_transition on segment {0}'.format(current_seg_id))
```

Comment:
This error message on Postgres would show up with `segment 0`. Does Postgres officially have segment in its terminology? If not, we should rephrase this.